test/plugins/individual-collateral/stargate/StargateUSDbCTestSuite.test.ts

```diff
@@ -0,0 +1,278 @@
+import collateralTests from '../collateralTests'
```

Comment:
Is it necessary to duplicate this whole file just ot add `targetNetwork`? Seems like an overkill. Cant it work with the same file for both networks?